### PR TITLE
Add quick rename button

### PR DESCRIPTION
**Pull request by lookacat**
## Description
See https://github.com/owncloud/web/issues/6626

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/src/components/FilesList/ResourceTable.vue [hunk 1]
```diff
@@ -45,17 +45,27 @@
...
+        <oc-resource
+          :key="`${item.path}-${resourceDomSelector(item)}-${item.thumbnail}`"
+          :resource="item"
+          :is-path-displayed="arePathsDisplayed"
+          :parent-folder-name-default="defaultParentFolderName"
+          :is-thumbnail-displayed="areThumbnailsDisplayed"
+          :is-resource-clickable="isResourceClickable(item.id)"
+          :folder-link="folderLink(item)"
+          :parent-folder-link="parentFolderLink(item)"
+          class=""
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/FilesList/ResourceTable.vue:58]:
```suggestion
```

Line can be removed

File: packages/web-app-files/src/components/FilesList/ResourceTable.vue [hunk 2]
```diff
@@ -648,6 +663,16 @@ export default {
...
 <style lang="scss">
 .resource-table {
+  &-resource-wrapper {
+    &:hover > .resource-table-edit-name {
+      opacity: 1;
+    }
+  }
+  &-edit-name {
+    display: inline-flex;
+    vertical-align: super;
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/FilesList/ResourceTable.vue:673]: Not sure if this is necessary since we're using flexbox?

Comment (lookacat) [reply packages/web-app-files/src/components/FilesList/ResourceTable.vue:673]: tried to do it without it and didn't center vertically, no idea why the other div for the resource is centered fine :/

File: packages/web-app-files/src/components/FilesList/ResourceTable.vue [hunk 3]
```diff
@@ -45,17 +45,27 @@
...
+          :key="`${item.path}-${resourceDomSelector(item)}-${item.thumbnail}`"
+          :resource="item"
+          :is-path-displayed="arePathsDisplayed"
+          :parent-folder-name-default="defaultParentFolderName"
+          :is-thumbnail-displayed="areThumbnailsDisplayed"
+          :is-resource-clickable="isResourceClickable(item.id)"
+          :folder-link="folderLink(item)"
+          :parent-folder-link="parentFolderLink(item)"
+          class=""
+          style="display: inline-flex"
```
[the remark this answers is not in the document]
Comment (lookacat) [reply packages/web-app-files/src/components/FilesList/ResourceTable.vue:59]: sorry dev leftover will remove it

**Pull request closed without merging**